src/db/contractEventLogs/createContractEventLogs.ts

```diff
@@ -0,0 +1,35 @@
+import { PrismaTransaction } from "../../schema/prisma";
+import { getPrismaWithPostgresTx } from "../client";
+
+export interface ContractEventLogEntry {
+  chainId: number;
+  blockNumber: number;
+  contractAddress: string;
+  transactionHash: string;
+  topic0?: string;
+  topic1?: string;
+  topic2?: string;
+  topic3?: string;
+  data: string;
+  decodedLog?: any; // Assuming JSON object for decodedLog
+  eventName?: string;
+  timestamp: Date;
+  transactionIndex: number;
+  logIndex: number;
+}
+
+export interface BulkInsertContractLogsParams {
+  pgtx?: PrismaTransaction;
+  logs: ContractEventLogEntry[];
```

Comment:
maybe we can use the Prisma Table Schema definition which should already be available and then we do not need to explicitly define this interface.

if we want to use this interface and mark it as `export` then we should look into adding into `sharedSchemas.ts` file.

Reply:
makes sense. Can change to use generated schema